src/Discussion/DiscussionRepository.php

```diff
@@ -54,6 +56,20 @@ public function getReadIds(User $user)
             ->all();
```

Comment:
We could implement this via the replacement for reduced code duplication.

Reply:
Absolutely. done!